.gitignore

```diff
@@ -1,12 +1,11 @@
-tmp/
 
 !dist/
-
 dist/*
-!dist/img/
```

Comment:
These guys is there to make sure, that you are not committing anything you shouldn't. We only want images, main.js and main.css